src/cli-logger.ts

```diff
@@ -0,0 +1,36 @@
+import chalk from "chalk";
+
+export const warning = chalk.bgYellowBright.whiteBright;
+export const info = chalk.bgBlueBright.whiteBright;
+export const success = chalk.green;
+export const secondary = chalk.gray;
+
+export function error(str: string, indentationLevel = 0) {
+  const err = chalk.red;
+  let prefix = "";
+  for (let i = 0; i < indentationLevel; i++) {
+    prefix += "--";
+  }
+  if (prefix) {
+    prefix += "> ";
+  }
+  // eslint-disable-next-line no-console
+  console.error(secondary(prefix) + err.inverse("ERROR") + err(` ${str}`));
+}
+
+export function log(str: string, indentationLevel = 0) {
```

Comment:
Why is there a `log` function manually implemented without chalk, but then also several chalk logging functions being exported?

Reply:
the log is only logging out in console, `warning` `info` `secondary` are chalk instances exported to style what is being logged such as:
```js
log(
    info(
      `Creating microfrontend ${info.inverse(name)}` +
        ` within ${info.inverse(useCustomerOrgKeyAsScope ? orgKey : scope)} scope`,
    ),
  );
```